schemas/json/block.json

```diff
@@ -26,6 +26,17 @@
 			"pattern": "^[a-z][a-z0-9-]*\/[a-z][a-z0-9-]*$",
 			"description": "The name for a block is a unique string that identifies a block. Names have to be structured as `namespace/block-name`, where namespace is the name of your plugin or theme."
 		},
+		"__experimental": {
```

Comment:
> Technically speaking we could rename it __experiment 

I thought about that and decided __experimental is more useful:

* Consistency: Gutenberg uses `__experimental_ in other places
* Searchability: It's easy to find this by grepping
* Brownie points: it will come up in experimental APIs audits that look for the existing prefix

> Make string only

I started this way, but there's a problem with that approach. Note that we have different experiment names such as `fse` or `list`, but we also have blocks that should always be enabled in the plugin. This means we need some special token. Initially, it was yes/no, but then I realized this option could be either:

* Boolean true – to say _always on_ in the plugin, or
* A string – when a feature flag is needed

We may be already good here, I'll just remove any leftovers from the version where "no" had special semantics.

Reply:
Works for me, too many great benefits confirming your choice 🤣